src/Security/Authentication/OAuth/src/MapAllClaimsAction.cs

```diff
@@ -13,10 +13,14 @@ namespace Microsoft.AspNetCore.Authentication.OAuth.Claims
     /// </summary>
     public class MapAllClaimsAction : ClaimAction
     {
+        /// <summary>
+        /// Initializes a new instance of <see cref="MapAllClaimsAction"/>.
```

Comment:
Why do you use See when referring to the current type?

Reply:
It makes it hyperlink back to the type in the docs. Also makes it so that the doc can be refactored if the type is renamed

Reply:
We're not suppose to rename types Pranav 😁